REVIEWING.md

```diff
@@ -0,0 +1,115 @@
+# Reviewing Pull Requests
+
+This document contains general principles for how to perform code reviews in the Podman repository.
+It does not aim to be a complete guide to how to perform code review, but rather to provide general guidance on how code reviews should be performed.
+
+This document is aimed at Reviewers, Maintainers, and Core Maintainers (see [GOVERNANCE.md](./GOVERNANCE.md) for definitions of these roles), but these guidelines should be followed by all who wish to review code in the Podman project's Github repositories, even those who are not currently a maintainer.
+
+## How are reviews performed
+
+The Podman project aims to ensure that all PRs are reviewed by at least 2 people prior to merge, at least one of which must be a repository Maintainer.
+There are some exceptions to this: Updates to libraries (including Go vendor updates) that pass CI cleanly and require no code changes may be merged by a maintainer without further review.
+
+All code merged must pass CI.
+
+## What should you review?
+
+We encourage review of all PRs, even those not in the area of expertise of the reviewer.
+Even if you are not fully familiar with the code in question, you can still notice basic mistakes.
+Feel free to ask questions about how areas of the code work to help familiarize yourself during reviews.
+If you finish a review and do not feel like you adequately understood the code to approve it for merge, tag an expert in that area of the code to perform a further review.
+
+Timely PR reviews are important - contributors can become discouraged if a PR is neglected by maintainers.
+All Maintainers and Reviewers should try to review new pull requests in their repositories once a day to ensure this.
+When you review a PR with failing tests, please check to see if those tests failed due to known flakes.
+If so, please restart the failed tests.
+Many repositories in the Podman project can only have their tests restarted by project members, not the submitter, so regular attention to test failures by reviewers is important.
+
+## Things to Check
+
+### Breaking Changes
+
+The Podman project aims to present a stable API for its users.
+Breaking changes to the project's Command Line Interfaces or public APIs (the Podman REST API and its associated bindings) must only be made in approved breaking change releases - Podman 6.0, 7.0, etc.
```

Comment:
I think there are precedents for exceptions that should possibly be noted OR a more general statement saying exceptions may be made by whatever/whoever.  Things I am thinking of:

* correction of a bug where it was implemented incorrectly . Example would be docker compat
* security related
* altering of defaults specifically for the benefit of users

Reply:
That's an excellent point, especially the security angle....

Reply:
If we had to do that, it would have to be Podman 6.0. We don't do breaking changes in 5.x, even for security.